### ISSUE TITLE
hyperref option clash error in recent texlive:latest image

### ISSUE THREAD

**Issue by hoodmane** [ho-tex/hypdoc#2]
This is probably already resolved. I figured I would report it anyways. 

My CI job is showing an option clash from this package in the docker image `texlive/texlive@sha256:b8fd3da6d8061d130c029df65b0a332f30ee1dd4a5f8b995837915374cfe479d`.

<details>
<summary>Details</summary>

```tex
(/usr/local/texlive/2022/texmf-dist/tex/latex/hypdoc/hypdoc.sty

! LaTeX Error: Option clash for package hyperref.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.54 ]{hyperref}[2002/05/09]
                            %
```
</details>

The full CI job:
https://app.circleci.com/pipelines/github/SpectralSequences/latex/132/workflows/38971b14-db8e-4838-9ed0-8478ccb28426/jobs/1848

In the current `texlive/texlive:latest` image `sha256:74dd345c635adb0b0159d768aa26d1dbc2c5c69748dd0341b38073eddae3be4d84` the problem is gone:
https://app.circleci.com/pipelines/github/SpectralSequences/latex/132/workflows/3706fad7-392f-4181-8e15-f5dcd13c590b/jobs/1851

**Comment by u-fischer** [comment]
yes that has been probably resolved. In any case: when reporting an issue make a small complete example and add it to the issue together with the log-file.
